config/qt5_core.mpb

```diff
@@ -0,0 +1,56 @@
+project {
+  expand(QT5_INCDIR) {
+    $QT5_INCDIR
+    $(QT5_DIR)/include
+  }
+
+  requires += qt5
+  includes += $(QT5_INCDIR)
+  libpaths += $(QT5_DIR)/lib
+  libs += Qt5Core
+
+  // On Linux at least, Qt5 by default requires position independent code
+  verbatim (make, macros) {
```

Comment:
Shouldn't both verbatims have an additional ,1 to not overwrite any other verbatim?

Reply:
That's true.  I was trying to see if we could avoid this part, but unfortunately we may be stuck.  What seems to be happening is that the Qt installed by Linux package managers has QT_REDUCE_RELOCATIONS defined.  This places requirements on applications that link these libraries.  (One more example why Linux package managers and global root-owned system installs are not a good fit for development libraries.)